quarkus-picocli/src/main/java/io/quarkus/ts/qe/configuration/Config.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.ts.qe.configuration;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Produces;
+
+import io.quarkus.arc.profile.IfBuildProfile;
+import io.quarkus.picocli.runtime.annotations.TopCommand;
+import io.quarkus.ts.qe.command.EntryCommand;
+import io.quarkus.ts.qe.command.OtherEntryCommand;
+
+@ApplicationScoped
+public class Config {
+    @Produces
+    @TopCommand
+    @IfBuildProfile("dev")
+    public Object devCommand() {
```

Comment:
`Object`?

Reply:
As in the guide example: https://quarkus.io/guides/picocli#different-entry-command-for-each-profile